### PR TITLE
Improve the time resolved spectroscopy tutorial

### PR DESCRIPTION
**Pull request by cgalelli**
This PR improves the time-resolved spectroscopy tutorial by:

-Removing the "aim" section title as discussed in the last dev call
-Improving the hysteresis plot by adding errors on amplitude and index and adding the label on the colorbar

### CODE REVIEW

**Review by AtreyeeS — DISMISSED** [review]
Thanks for the fix @Astro-Kirsty

**Review by bkhelifi — APPROVED** [review]
Thank you a lot

File: examples/tutorials/analysis-time/Time_resolved_spectroscopy.py [hunk 1]
```diff
@@ -1,10 +1,7 @@
 """
-A time resolved spectroscopy estimator
+Time resolved spectroscopy estimator
 ======================================
```
Comment (Astro-Kirsty) [inline examples/tutorials/analysis-time/Time_resolved_spectroscopy.py:3]:
```suggestion
Time resolved spectroscopy estimator
====================================
```

**Review by Astro-Kirsty — COMMENTED** [review]
Thanks @cgalelli

**Review by Astro-Kirsty — APPROVED** [review]

**Review by registerrier — APPROVED** [review]
Thanks @cgalelli !

**Pull request merged**